src/Vatsim/Provider.php

```diff
@@ -76,9 +76,17 @@ protected function getUserByToken($token)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'    => Arr::get($user, 'data.cid'),
-            'name'  => Arr::get($user, 'data.personal.name_full'),
-            'email' => Arr::get($user, 'data.personal.email'),
+            'cid'    => Arr::get($user, 'data.cid'),
+            'first_name' => Arr::get($user, 'data.personal.name_first'),
+            'last_name' => Arr::get($user, 'data.personal.name_last'),
+            'full_name'  => Arr::get($user, 'data.personal.name_full'),
+            'rating' => Arr::get($user, 'data.vatsim.rating.id'),
+            'pilot_rating' => Arr::get($user, 'data.vatsim.pilotrating.id'),
+            'region' => Arr::get($user, 'data.vatsim.region.id'),
```

Comment:
Please update the `README` to have the returned fields :)

Reply:
Done. Sorry for the delay.